explainer/index.html

```diff
@@ -0,0 +1,553 @@
+            <ul><li>Bronze, Silver and Gold conformance levels: AG has considered using three traditionally named conformance levels. 
+                <ul><li>Bronze would be the minimum conformance level. Content that does not meet the requirements of the
+                    bronze level does not conform to WCAG 3.0. The subset will require enough requirements and
+                    assertions to improve <a>equity</a> across functional needs.</li>
+                <li>Silver level incentivizes organizations to go further to improve accessibility. The goal is to encourage 
+                    organizations to go beyond the minimum, especially where organizations want to be recognized for their efforts to go beyond minimum
+                    accessibility.</li>
+                <li>Gold level identifies measures we want to include for those organizations that do achieve
+                    Silver so that some can stand out as exemplary, cutting edge, and role models.</li></ul>    
+                <li>Issue severity: Severity ratings could contribute towards scoring and prioritization.</li>
+                <li>Adjectival ratings: Adjectival Ratings allow test results to go beyond Pass or Fail to show progress towards a goal or
+                    exceeding a goal. Example of possible adjectival ratings are:
+                <ul>
+                    <li>Fail, Pass, Exceptional; or</li>
+                    <li>Fail, Progress, Pass, Better, Exceptional.</li>
+                </ul>
+                <li>Pre-assessment checks: Pre-Assessment checks are tests or criteria that implementers can use to determine if they are ready to
+                    assess conformance. The intent of specifying these would be to help implementers prepare for conformance
+                    testing, not to create a new level of conformance. </li>
+            </ul>   
+    </section>
+
+    <section id="schedule">
+        <h2>Schedule</h2>
+        <p>AG is currently <a href="https://www.w3.org/2023/11/ag-charter">chartered</a> through November 2025. The
+            schedule through that time period is maintained at <a
+                href="https://www.w3.org/WAI/GL/wiki/WCAG_3_Timeline#Publication_Plan">WCAG 3.0 Timeline</a>.</p>
+        <p>AG will continue to iterate within a single normative document with informative supporting documentation.
+            After exploring other options such as breaking the content's into modules, the group determined that the
+            fastest way forward at this time is to continue creating a single document with varying levels of maturity.
+            We will revisit whether mature content can be published as informative modules as part of the next charter
+            period, when content is more mature.</p>
+        <p>A final schedule will be delivered as part of this charter period. WCAG 3.0 will not be published until it
+            covers at least as much as WCAG 2.2.</p>
```

Comment:
Suggest "coverage comparable to WCAG 2.2 Level AA" and that the phrasing be consistent with the reference in 7. Draft Conformance Approach.

Reply:
Not discussed in meeting of 2024-11-05. Resolving with no change but @bruce-usab please do raise an issue once I have merged this in if you still have concerns.